onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -1302,6 +1338,7 @@ common::Status TensorrtExecutionProvider::Compile(const std::vector<Node*>& fuse
     if (!has_dynamic_shape) {
       const std::string cache_path = GetCachePath(cache_path_, trt_node_name_with_precision);
       const std::string engine_cache_path = cache_path + ".engine";
+      const std::string timing_cache_path = cache_path + ".timing";
```

Comment:
i'm wondering if we should put the timing cache in the same location as engine cache/profiles 
the engines may be model specific, but timing cache can be system wide?